crates/lib/src/compiler/quilc.rs

```diff
@@ -17,6 +16,18 @@ use crate::client::Qcs;
 /// Number of seconds to wait before timing out.
 pub const DEFAULT_COMPILER_TIMEOUT: f64 = 30.0;
 
+/// The Quilc compiler methods
+pub trait Client {
+    /// Compile the program `quil` for the given target device `isa`
+    /// with the compilation options `options`.
+    fn compile_program(
```

Comment:
Whats the plan for the other quilc requests? Will those be supported by `libquil`?

Reply:
Yes! Working on it.